docs/getting-started/integration.md

```diff
@@ -96,6 +89,16 @@ And finally there is a separate path to do just the above for you, in one line:
 import Chart from 'chart.js/auto';
 ```
 
+## CommonJS
+
+Due to Chart.js 4 is ESM-only, in CommonJS modules you should use dynamic `import`:
```

Comment:
I'm not sure it's necessary to mention the version number. It's just one more thing to update for Chart.js 5 one day. E.g. we should probably remove the version number from line 16 which will be out-of-date with this new release. Also, with just a couple minor grammar suggestions, I might write it as:
> Because Chart.js is an ESM library, in CommonJS modules you should use a dynamic `import`: